amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
I'm not against first; also I don't quite understand the second option :P

Just suggesting keeping the binary format in mind for design. I actually like the first more (except for perhaps embedding the uuid, this closely couples panopticon's uuid implementation with the rreil, which is fine, but something to be aware of)

Another approach is to consider lifting the "primitive" rreil IL into an IR, which rewrites and contains all these analyses after the fact.

If we did this, we can let the rreil be dumb, and easier to encode, easier to emit, and then have a second pass take the IL as input, and output a more sophisticated IR, which contains stuff like resolved call functions with read/writes, etc., which can then be taken as input to something like a decompiler backend